stable/satisfy/values.yaml

```diff
@@ -0,0 +1,71 @@
+# Default values for satisfy.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: docker.io/anapsix/satisfy
+  # if both image.tag and image.digest are present
+  # digest will be used
+  tag: v3.0.4
+  digest: sha256:16a818ab03b6a6bd2625dc9b4760f079fbbddfdd1473994593dfae958c6e7e29
```

Comment:
Don't specify both, tag and digest. Only one is used anyways. The digest can be used as tag.

Reply:
Since Docker demanded different syntax for digest vs. tag pulls (`imageName@imageDigest` vs. `imageName:imageTag`) I'm using digest explicitly
I've tested with K8s version 1.12, and it expects same syntax as Docker (`imageName@imageDigest`), when using image digest